.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ on:
   pull_request:
   push:
     branches:
-      - main
+      - develop
```

Comment:
this should fix Badges for Flags https://github.com/codecov/feedback/issues/308#issuecomment-2018248760